### ISSUE TITLE
⛔ ♿ Accessibility: [aria-hidden="true"] elements contain focusable descendants in the WordPress Menu widget

### ISSUE THREAD

**Issue by jazir555** [elementor/elementor#28035]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Lighthouse warning for [aria-hidden="true"] elements contain focusable descendants.

![Screenshot 2024-07-11 224356](https://github.com/user-attachments/assets/d3716fdd-e8ce-41b5-82e2-4f6e5eafc1bf)

### Steps to reproduce

1. Build a menu
2. Test a page with lighthouse

### Expected behavior

No error

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

```txt
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.24.0
	MySQL version: mariadb.org binary distribution v10.11.7-MariaDB-1:10.11.7+maria~deb12-log
	PHP Version: 8.3.6
	PHP Memory Limit: 256M
	PHP Max Input Vars: 20000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.4
	Site URL: https://strongbpc.com
	Home URL: https://strongbpc.com
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 1024M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: -8

	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.0.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Database Cleaner PRO
		Version: 3.2.8
		Author: Younes JFR.

	Asset CleanUp Pro: Page Speed Booster
		Version: 1.2.4.9
		Author: Gabe Livan

	Disable Gutenberg
		Version: 3.1.1
		Author: Jeff Starr

	Docket Cache
		Version: 23.08.02
		Author: Nawawi Jamili

	Duplicate Page
		Version: 4.5.3
		Author: mndpsingh287

	Duplicator Pro
		Version: 4.5.16.3
		Author: Duplicator

	Elementor
		Version: 3.21.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.21.0
		Author: Elementor.com

	File Manager Advanced
		Version: 5.2.3
		Author: modalweb

	Fluent Snippets
		Version: 10.33
		Author: Fluent Snippets

	Flying Pages
		Version: 2.4.6
		Author: WP Speed Matters

	FlyingPress
		Version: 4.13.3
		Author: FlyingWeb

	Hide admin notices
		Version: 3.1.0
		Author: Rémi Leclercq

	Minify HTML
		Version: 2.1.9
		Author: Tim Eckel

	Perfmatters
		Version: 2.2.6
		Author: forgemedia

	Rank Math SEO
		Version: 1.0.216
		Author: Rank Math

	Show IDs by Echo
		Version: 1.1.3
		Author: Echo Plugins

	Stream
		Version: 4.0.0
		Author: XWP

	WooCommerce
		Version: 8.7.0
		Author: Automattic

	WooCommerce Cost of Goods
		Version: 2.13.0
		Author: SkyVerge

	WPCode Lite
		Version: 2.1.11
		Author: WPCode

	WPO Tweaks &amp; Optimizations
		Version: 1.0.4
		Author: Fernando Tellado

	WP Tweaks
		Version: 2.3.4
		Author: Luiz Bills


== Must-Use Plugins ==
	Asset CleanUp Pro: Plugin Filtering
		Version: 1.0
		Author: Gabriel Livan


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	woocommerce: Active


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Upgrade Swiper Library: Active by default
	Grid Container: Active by default
	Nested Elements Performance: Inactive by default
	Optimized Control Loading: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Inactive
	Elementor Home Screen: Active by default
	Editor Top Bar: Active
	Build with AI: Active by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Pages Panel: Inactive by default
	Display Conditions: Inactive by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
Log: showing 20 of 312024-03-15 22:14:09 [info] elementor-pro::elementor_pro_updater Started 
2024-03-15 22:14:09 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-03-15 22:14:09 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-03-15 22:14:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.3',
  'to' => '3.20.0',
)]
2024-03-15 22:14:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.3',
  'to' => '3.20.0',
)]
2024-03-15 22:14:09 [info] elementor::elementor_updater Started 
2024-03-15 22:14:09 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-15 22:14:09 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-15 22:14:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.4',
  'to' => '3.20.0',
)]
2024-03-15 22:14:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.4',
  'to' => '3.20.0',
)]
2024-03-15 22:14:10 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.3',
  'to' => '3.20.0',
)]
2024-03-15 22:15:09 [info] elementor::elementor_updater Started 
2024-03-15 22:15:09 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-15 22:15:09 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-15 22:15:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]
2024-04-15 13:57:27 [info] elementor::elementor_updater Started 
2024-04-15 13:57:27 [info] Elementor/Upgrades - _on_each_version Start  
2024-04-15 13:57:27 [info] Elementor/Upgrades - _on_each_version Finished 
2024-04-15 13:57:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.1',
  'to' => '3.21.0',
)]
2024-04-15 13:57:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.1',
  'to' => '3.21.0',
)]

PHP: showing 6 of 6PHP: 2024-02-21 01:58:52 [notice X 104][../wp-content/plugins/elementor-pro/modules/woocommerce/module.php::626] parse_str(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-02-21 16:28:23 [notice X 3][../wp-content/plugins/elementor/includes/api.php::164] Automatic conversion of false to array is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-02-21 16:37:40 [warning X 32][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0:../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ./wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-03-22 22:33:05 [warning X 8][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::179] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-04-15 13:57:27 [warning X 1][../wp-content/plugins/elementor/core/files/manager.php::111] unlink(../wp-content/uploads/elementor/css/custom-widget-icon-box.min.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-04-15 13:57:27 [warning X 1][../wp-content/plugins/elementor/core/files/manager.php::111] unlink(../wp-content/uploads/elementor/css/custom-widget-icon-list.min.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2024-02-22 00:11:46 [error X 24][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.2:3:575963] Failed to read a named property &#039;elementorFrontend&#039; from &#039;Window&#039;: Blocked a frame with origin &quot;.. ; from accessing a cross-origin frame. 
JS: 2024-02-22 00:32:28 [error X 14][.. /wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.19.2:2:64496] Cannot convert undefined or null to object 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi. This is a false positive. The items is question are inside an element that has display: none. Which means they are hidden as well. The aria-hidden changes to false when you open the dropdown, which shows the focus able content inside.

You can ignore the lighthouse warning.
